subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/EnableFileSystemWatchingIntegrationTest.groovy

```diff
@@ -17,11 +17,16 @@
 package org.gradle.internal.watch
 
 import org.gradle.initialization.StartParameterBuildOptions
+import org.gradle.internal.watch.vfs.WatchMode
 import spock.lang.Unroll
 
 class EnableFileSystemWatchingIntegrationTest extends AbstractFileSystemWatchingIntegrationTest {
-    private static final String ENABLED_MESSAGE = "Watching the file system is enabled"
-    private static final String DISABLED_MESSAGE = "Watching the file system is disabled"
+    private static final String ENABLED_MESSAGE = "Watching the file system is configured to be enabled"
+    private static final String ENABLED_IF_AVAILABLE_MESSAGE = "Watching the file system is configured to be enabled if available"
+    private static final String DISABLED_MESSAGE = "Watching the file system is configured to be disabled"
+
+    private static final String ACTIVE_MESSAGE = "File system watching is active"
+    private static final String INACTIVE_MESSAGE = "File system watching is inactive"
```

Comment:
If file system watching is enabled, but not supported on the OS, we will be printing an extra message at the warn level:
```
Watching the file system is not supported on this operating system.
```

Should we change this message as well, so it is clear watching has been requested explicitly/

Reply:
How would you change it?

Reply:
In my other PR, I changed the message to `Watching the file system is not supported.`.

I wondered if we should use something like `File system watching has been requested, but is not supported.`. We can also leave this as is.